typescript/infra/config/environments/mainnet3/warp/strategies/ousdt.yaml

```diff
@@ -2,31 +2,42 @@ base:
   submitter:
     chain: base
     safeAddress: '0x125d1b64dfd7898DD06ac3E060A432691b8Fa676'
-    type: gnosisSafe
+    type: gnosisSafeTxBuilder
```

Comment:
Shouldn't this all be timelock submitter? Or is that because that's not ready yet?

Reply:
Or at minimum ICA?